float64 height
float64 threshold

---

bool success